### DISCUSSION TITLE
Document use case in Relay docs?

### DISCUSSION THREAD

**Discussion by artola** [enisdenjo/graphql-ws#304] [Ideas]
It is a pity to see in the Relay documentation for subscriptions just ["other" library](https://relay.dev/docs/guided-tour/updating-data/graphql-subscriptions/#configuring-the-network-layer) as example of implementation.
It would be nice to enhance their documentation with `graphql-ws`.

**Comment by enisdenjo** [comment]
I've already made a PR there (https://github.com/facebook/relay/pull/3245), almost 2 years ago. 😅

**Reply by artola** [reply]
Thanks @enisdenjo. Heads up: https://github.com/facebook/relay/pull/3245#issuecomment-1033632631

**Reply by artola** [reply]
Congrats @enisdenjo !!! 🚀 🚀 🚀 

It is online: https://relay.dev/docs/next/guided-tour/updating-data/graphql-subscriptions/#configuring-the-network-layer

**Reply by artola** [reply]
Now looking for @michaelstaib to add support into [HotChocolate](https://chillicream.com/docs/hotchocolate).
